lib/sql/migrations.dart

```diff
@@ -1,16 +1,17 @@
 import 'package:sqflite/sqflite.dart';
 import 'package:twake/sql/v8.dart';
+import 'package:twake/sql/v9.dart';
 
 import 'v6.dart';
 
 const CURRENT_MIGRATION = DDL_V6;
-const DBVER = DB_V8;
+const DBVER = DB_V9;
 
 Future<void> dbUpgrade({required Database db, required int oldVersion, required int newVersion}) async {
   if (newVersion > oldVersion) {
-    if(oldVersion < DB_V8) {
+    if(oldVersion < DB_V9) {
```

Comment:
In case user has not upgraded into V8 yet (still be <= V7), they may miss V8 changes. Need to have more checking here, for eg:

```
final batch = db.batch();
if(oldVersion < DB_V8) {
  MIGRATION_8.forEach((element) {
    batch.execute(element);
  });
} else if(oldVersion < DB_V9) {
  MIGRATION_9.forEach((element) {
    batch.execute(element);
  });
}
await batch.commit();
```